tests/integration.rs

```diff
@@ -4356,6 +4356,97 @@ mod tests {
             handle_child_output(r, &output);
         }
 
+        fn get_balloon_mem(api_socket: &str) -> u64 {
+            let (cmd_success, cmd_output) = remote_command_w_output(&api_socket, "info", None);
+            assert!(cmd_success);
+
+            let info: serde_json::Value = serde_json::from_slice(&cmd_output).unwrap_or_default();
+            let total_mem = &info["config"]["memory"]["size"]
+                .to_string()
+                .parse::<u64>()
+                .unwrap();
+            let actual_mem = &info["memory_actual_size"]
+                .to_string()
+                .parse::<u64>()
+                .unwrap();
+            total_mem - actual_mem
+        }
+
+        #[test]
+        #[cfg(target_arch = "x86_64")]
+        fn test_virtio_balloon() {
+            let focal = UbuntuDiskConfig::new(FOCAL_IMAGE_NAME.to_string());
+            let guest = Guest::new(Box::new(focal));
+            let mut workload_path = dirs::home_dir().unwrap();
+            workload_path.push("workloads");
+
+            let kernel_path = direct_kernel_boot_path();
+
+            let api_socket = temp_api_path(&guest.tmp_dir);
+
+            let guest_memory_size_gb = 32;
```

Comment:
Can you use smaller sizes? I mean maybe you could try to run a 4G VM with a balloon of 2G. Do you think that would be reasonable?
The reason I'm asking is because it'd be faster to consume the memory from a guest perspective, and it would also have less potential impact on other tests running in parallel on the host.

Reply:
> Can you use smaller sizes? I mean maybe you could try to run a 4G VM with a balloon of 2G. Do you think that would be reasonable?
> The reason I'm asking is because it'd be faster to consume the memory from a guest perspective, and it would also have less potential impact on other tests running in parallel on the host.

Sure.